File: AssetManagement.Application/Employees/GetEmployees/GetEmployeesResponse.cs
namespace AssetManagement.Application.Employees.GetEmployees;

public sealed record GetEmployeesResponse(
    string EmployeeNumber,
    string FullName,
    DateTime EmploymentStartDate,
    DateTime? EmploymentEndDate,
    string? StoreNumber,
    string OrganizationType,
    string PositionCode,
    string Position,
    string? PhoneNumber,
    string? Email);

File: Client/src/app/features/employees/data-access/models/get-employees-response.ts
export interface GetEmployeesResponse {
  readonly employeeNumber: string;
  readonly fullName: string;
  readonly employmentStartDate: string;
  readonly employmentEndDate: string | null;
  readonly storeNumber: string;
  readonly organizationType: string;
  readonly positionCode: string;
  readonly position: string;
  readonly phoneNumber: string | null;
  readonly email: string | null;
}
